app/controllers/admin_controller.rb

```diff
@@ -382,7 +382,7 @@ def do_override_regional_records
   end
 
   def all_voters
-    voters User.eligible_voters, "all-wca-voters"
+    voters Api::V0::UserRolesController.eligible_voters, "all-wca-voters"
```

Comment:
I am not happy about calling controllers from other controllers.
Is there a reasonable place in `app/models` or in `lib` where you can store this code?

Reply:
Maybe I'll have it in `user` model itself for now. I was thinking to make it as an API for the react migration, but will do that later.